lib/utils/other/size_meassure.dart

```diff
@@ -16,7 +16,6 @@ import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 
 class SizeMeasureWidget extends SingleChildRenderObjectWidget {
-  // ignore: annotate_overrides, overridden_fields
   final Widget child;
```

Comment:
yea ok remove the ignore and also remove the declaration here, it will be handled by the super class